crates/agglayer-types/src/lib.rs

```diff
@@ -13,7 +13,7 @@ pub mod network_info;
 mod proof_modes;
 
 #[cfg(feature = "testutils")]
-pub use certificate::compute_signature_info;
+pub use certificate::{compute_signature_info, EMPTY_ELF};
```

Comment:
Nit: May be cleaner to keep these under testutils namespace. Tests will need to be adjusted for this to work though.
```suggestion
pub mod testutils {
    pub use certificate::{compute_signature_info, EMPTY_ELF};
}
```